BridgeCareApp/BridgeCare/Security/JWTParse.cs

```diff
@@ -146,7 +149,7 @@ private static List<string> ParseLDAP(string ldap)
             List<string> commonNames = new List<string>();
             foreach (string segment in segments)
             {
-                string firstSubSegment = ldap.Split(',')[0];
+                string firstSubSegment = segment.Split(',')[0];
```

Comment:
This still does not address the issue of getting a "roles" value with an empty or null string.  A null string will crash here (because String.Split(null) will return a null reference error), while an empty string will crash below because "=" won't be found so element 1 will not be found.  Please address.

Reply:
A null or empty roles string will now correctly produce an empty list of roles, which will produce the appropriate authentication exception.